packages/pf-codemods/index.js

```diff
@@ -2,6 +2,7 @@
 const path = require('path');
 const options = require('eslint/lib/options');
 const { CLIEngine } = require('eslint/lib/cli-engine');
+const lodash = require('lodash');
```

Comment:
It's possible `lodash` was installed via another package (e.g., Victory or react-charts), but should add `lodash` as a `pf-codemods` dependency.

Reply:
I removed the need to use lodash as it is most likely over kill for our needs here.